test/test_jit.py

```diff
@@ -5001,6 +5001,106 @@ def test(op, const, swap_args):
     def test_tensor_number_math(self):
         self._test_tensor_number_math()
 
+    def test_torch_tensor_bad_input(self):
+        with self.assertRaisesRegex(RuntimeError, "Input list to torch.tensor must be of ints, floats, "
+                                    "or bools, got None"):
+            @torch.jit.script
+            def test():
+                return torch.tensor([None])
+
+        with self.assertRaisesRegex(RuntimeError, "Note: empty lists are constructed as Tensor"):
```

Comment:
What does this error message mean?

Reply:
It's the same error message as in the compiler:
```
empty lists are constructed as Tensor[]; \n"
               << "if you want an empty list of a different type, \n"
               << "use `torch.jit.annotate(List[T], [])`, \n"
               << "where `T` is the type of elements in the list)";
```
We type empty lists as tensor lists by default. So if a user tries torch.tensor([]) the more informative error message is printed, in addition to `Input list to torch.tensor must be of ints, floats, or bools, go" << type->str().

Reply:
Got it, thanks for the explanation, I forgot that you only need to pass part of the error message to assertRaisesRegex